tests/serving/test_serving.py

```diff
@@ -529,6 +534,22 @@ def test_v2_get_modelmeta(rundb_mock):
         server.test("/v2/models/m4", method="GET")
 
 
+def test_v2_infer_dict(rundb_mock):
+    project = mlrun.new_project("tstsrv", save=False)
+    fn = mlrun.new_function("tst", kind="serving")
+    model_uri = _log_model(project)
+    print(model_uri)
+    fn.add_model("m1", model_uri, "ModelTestingClass", multiplier=100)
+
+    server = fn.to_mock_server()
+    resp_list = server.test("/v2/models/m1/infer", testdata_iris)
```

Comment:
Test also `/v2/models/m1/predict` - this will expose the issue I commented on earlier.